lib/bundle.js

```diff
@@ -74,11 +75,11 @@ module.exports = (targetBundle, cnfg, opts) => {
     })
   }
 
-  function bundle() {
+  function bundle(errorHandler) {
```

Comment:
@ryanerato This arg should actually be removed. I had started implementing in a way that needed that, but then changed tack. Will remove now.